hexrd/ui/indexing/run.py

```diff
@@ -141,15 +130,16 @@ def run_grain_fitting(self):
             'radius': config.find_orientations.clustering.radius
         }
         qbar, cl = run_cluster(**kwargs)
+        print('qbar:', qbar.shape)
+        print(qbar)
+        print('cl:', cl.shape)
+        print(cl)
 
         self.qbar = qbar
         self.cl = cl
         print('Grain fitting is complete!')
         print(f'{self.qbar.shape[1]} grains were found')
 
-    def fit_grains_config_finished(self, result):
-        # result 0 == rejected, 1 == accepted
-        if result == 1:
-            from PySide2.QtWidgets import QMessageBox
-            QMessageBox.information(
-                None, "Todo", "Sorry, Fit Grains not yet implemented")
+        from PySide2.QtWidgets import QMessageBox
```

Comment:
Is there a particular reason to do the import here? Otherwise lets move it to the top.

Reply:
Nope, no good reason for putting the import here. I'll force push a change to move it to the top.